static/css/materialize.css

```diff
@@ -1,8 +1,8 @@
-/*!

```

Comment:
I think these changes were made by accident in this file.  Could you revert them? :)